### PR TITLE
Remove extra `dpcpp` backend checks from `test/xpu_api` tests

### PR DESCRIPTION
**Pull request by SergeyKopienko**
In this PR we remove extra `dpcpp` backend checks from `test/xpu_api` tests.

This is the list of tests from which we remove this extra check:
The list of new added folders for skip tests in this case:
- `test/xpu_api/cmath/nearbyint`
- `test/xpu_api/containers`
- `test/xpu_api/iterators`
- `test/xpu_api/language.support/support.initlist`
- `test/xpu_api/language.support/support.types`
- `test/xpu_api/random`
- `test/xpu_api/ratio`
- `test/xpu_api/tuple`
- `test/xpu_api/utilities`

After PR https://github.com/oneapi-src/oneDPL/pull/1288 we already skip all tests from all these folders in [`test/CMakeList.txt`](https://github.com/oneapi-src/oneDPL/pull/1288/files#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fd) when backend is not `dpcpp`.

### CODE REVIEW

File: test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_compound.pass.cpp [hunk 1]
```diff
@@ -104,7 +104,8 @@ class KernelTest17;
 void
 kernel_test()
 {
-    sycl::queue deviceQueue = TestUtils::get_test_queue();;
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    ;
```
Comment (kboyarinov) [inline test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_compound.pass.cpp:108]:
```suggestion
    sycl::queue deviceQueue = TestUtils::get_test_queue();
```

File: test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_fundamental.pass.cpp [hunk 2]
```diff
@@ -116,7 +116,8 @@ class KernelTest31;
 void
 kernel_test()
 {
-    sycl::queue deviceQueue = TestUtils::get_test_queue();;
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    ;
```
Comment (kboyarinov) [inline test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_fundamental.pass.cpp:120]:
```suggestion
    sycl::queue deviceQueue = TestUtils::get_test_queue();
```

File: test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_member_pointer.pass.cpp [hunk 3]
```diff
@@ -114,7 +114,8 @@ void
 kernel_test()
 {
 
-    sycl::queue deviceQueue = TestUtils::get_test_queue();;
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    ;
```
Comment (kboyarinov) [inline test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_member_pointer.pass.cpp:118]:
```suggestion
    sycl::queue deviceQueue = TestUtils::get_test_queue();
```

File: test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_reference.pass.cpp [hunk 4]
```diff
@@ -107,7 +107,8 @@ class KernelTest20;
 void
 kernel_test()
 {
-    sycl::queue deviceQueue = TestUtils::get_test_queue();;
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    ;
```
Comment (kboyarinov) [inline test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_reference.pass.cpp:111]:
```suggestion
    sycl::queue deviceQueue = TestUtils::get_test_queue();
```

File: test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_scalar.pass.cpp [hunk 5]
```diff
@@ -116,7 +116,8 @@ class KernelTest29;
 void
 kernel_test()
 {
-    sycl::queue deviceQueue = TestUtils::get_test_queue();;
+    sycl::queue deviceQueue = TestUtils::get_test_queue();
+    ;
```
Comment (kboyarinov) [inline test/xpu_api/utilities/meta/meta.unary/meta.unary.comp/is_scalar.pass.cpp:120]:
```suggestion
    sycl::queue deviceQueue = TestUtils::get_test_queue();
```

File: test/xpu_api/utilities/utility/forward/move_if_noexcept.pass.cpp [hunk 6]
```diff
@@ -46,7 +45,7 @@ struct MoveNotNoexceptCopy
 {
     MoveNotNoexceptCopy() = default;
 
-    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false) {};
+    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false){};
```
Comment (kboyarinov) [inline test/xpu_api/utilities/utility/forward/move_if_noexcept.pass.cpp:48]: Please roll back this change
```suggestion
    MoveNotNoexceptCopy(MoveNotNoexceptCopy&&) noexcept(false) {};
```

File: test/xpu_api/language.support/support.initlist/support.initlist.cons/default_initlist.pass.cpp [hunk 7]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (SergeyKopienko) [reply test/xpu_api/language.support/support.initlist/support.initlist.cons/default_initlist.pass.cpp]: Done

**Pull request merged**